tests/garage/tf/algos/test_td3.py

```diff
@@ -71,4 +71,4 @@ def test_td3_pendulum(self):
 
             runner.setup(algo, env, sampler_cls=LocalSampler)
             last_avg_ret = runner.train(n_epochs=10, batch_size=250)
-            assert last_avg_ret > 300
+            assert last_avg_ret > 100
```

Comment:
Could it mean something is not working? What’s the average return before and after the change?

Reply:
I run 20 tests respectively in my local machine. The `last_avg_ret` is:
- Before change: 
mean = 425.04948958342857
std = 69.67050410993383
- After change:
mean = 430.92078589330924
std = 82.32598708139051
So I think this is not because this change.

Reply:
It seems really flaky. Is there already an issue for that? Can you set it to 200?

Reply:
In [last build](https://travis-ci.com/github/rlworkgroup/garage/jobs/362404060), it fails. The `last_avg_ret` is 151.178